models/user.js

```diff
@@ -172,12 +178,27 @@ UserSchema.methods.loginError = function() {
         message: "You are banned from using this service due to violations of the rules.",
         code: "banned"
     };
-    if (this.deactivated === true) return {
-        message: "Your account has been deactivated. Please contact the moderators via Discord to reactivate your account.",
-        code: "deactivated"
-    };
     return null;
 }
+
+// Helper
+Date.prototype.addDays = function(days) {
+    var dat = new Date(this.valueOf());
+    dat.setDate(dat.getDate() + days);
+    return dat;
```

Comment:
why is this inside user? also this code is so trivial, used only once in the whole project. it could just be done as part of `markForDeletion`

Reply:
This is actually harder than you’d think to get right, so I went with a proven method — sometimes inlining this can produce different results (strange date formatting etc)